main/src/vtr_navigation/src/graph_map_server.cpp

```diff
@@ -88,6 +89,8 @@ void GraphMapServer::start(const rclcpp::Node::SharedPtr& node,
   update_waypoint_sub_ = node->create_subscription<UpdateWaypointMsg>("update_waypoint", rclcpp::QoS(10), std::bind(&GraphMapServer::updateWaypointCallback, this, std::placeholders::_1), sub_opt);
   // clang-format on
 
+  pose_pub_ = node->create_subscription<NavSatFix>("/novatel/fix", rclcpp::QoS(10), std::bind(&GraphMapServer::poseCallback, this, std::placeholders::_1), sub_opt);
```

Comment:
Can we make the GPS topic a config file ros param to support other GPS units in the future

Reply:
Added